py/server/deephaven/parquet.py

```diff
@@ -76,30 +80,72 @@ def _build_parquet_instructions(
     if max_dictionary_size is not None:
         builder.setMaximumDictionarySize(max_dictionary_size)
 
-    builder.setIsLegacyParquet(is_legacy_parquet)
+    if is_legacy_parquet:
+        builder.setIsLegacyParquet(is_legacy_parquet)
 
     if target_page_size is not None:
         builder.setTargetPageSize(target_page_size)
 
-    builder.setIsRefreshing(is_refreshing)
+    if is_refreshing:
+        builder.setIsRefreshing(is_refreshing)
 
     return builder.build()
 
+def _j_table_definition(table_definition: Union[Dict[str, DType], List[Column], None]):
+    if table_definition is None:
+        return None
+    elif isinstance(table_definition, Dict):
+        return _JTableDefinition.of(
+            [
+                Column(name=name, data_type=dtype).j_column_definition
+                for name, dtype in table_definition.items()
+            ]
+        )
+    elif isinstance(table_definition, List):
+        return _JTableDefinition.of(
+            [col.j_column_definition for col in table_definition]
+        )
+    else:
+        raise DHError("not type")
+
+class ParquetType(Enum):
```

Comment:
Class name is not obvious for the user.  e.g. `ParqueFileLayout` would be more clear.